### ISSUE TITLE
How do stars die? (proposed key question)

### ISSUE THREAD

**Issue by ethrane** [ethrane/transients#1]
@ethrane and @kauchettl propose this as a key question. This question would cover topics like tidal disruption events, black holes + neutron stars, gravitational waves, supernovae, fast radio bursts in the context of stellar death, etc. It would also cover various topics from stellar remnants, e.g., pulsar-like emission from white dwarfs. There is good synergy with Group 1.2 (Stars, Planets, and the Galaxy). We like the idea of posing this question as a follow up to a question in the last decadal plan: "How do stars and planets form?"

Updated on 22 Feb from "How do _massive_ stars die?" to "How do stars die?"
Updated on 29 Apr to emphasise that the question is not just about the cataclysmic moment of death, but also remnants.

**Comment by kauchettl** [comment]
:)

**Comment by jeffcooke** [comment]
Either associated with this question, or a new questions is " What are the physics behind the most extreme explosions? This question addresses some of the events above, but also superluminous supernovae, weird repeating FBOTs, some of the very long lasting (1000s of days) events, etc. These are unknown and have been enigmatic and may point to new physics or phenomena (i.e., beyond better understanding events like most of those above).

**Comment by adellej** [comment]
I suggest an alternative: "How do stars die?" As this then broadens the science cases to include white dwarfs, type Ia sypernovae, most tidal disruption events etc.

**Comment by ethrane** [comment]
@adellej, I think there was broad support for your proposal, so I have implemented this suggestion and reworded the proposed key question.

**Comment by cwjames1983** [comment]
Thanks @adellej  @ethrane for this. Regarding @jeffcooke suggestion of "extreme explosions" - I think that's a new question. It obviously covers much of this, though not all (e.g. white dwarfs, and TDEs could kill stars without an "extreme explosion" perhaps), and as you say adds others (e.g. FRBs could clearly be in that one). Maybe @jeffcooke you could propose that question explicitly?

**Comment by jeffcooke** [comment]
We have a question that asks about the physics of matter at the most extreme densities, but not a question about the physics of the most extreme energies. Both are uniquely probed by transients. To help minimise question options, my question first aimed to first rephrase "How do stars die?" more to the latter. But I'm happy to propose this as a new question as well. Either way, the question would address the explosive deaths of stars, but also, importantly, extreme explosions (a fraction of these do not have stellar origins). They are the only place where we can learn about such extreme energies and would include high-energy particles (nature's particle accelerators) naturally. The extreme explosions I was referring to, beyond things like TDEs, include what powers LFBOTs (luminous fast blue optical transients), which are events that can burst with supernova energies in the optical - repeatedly(!) I.e., a supernova that is not cataclysmic. And, remarkably, on timescales of minutes (see the "tasmanian devil"). What can possibly do that? Also, superluminous supernovae are extreme (10-100x more luminous than supernovae) and it is unclear what powers them (magnetar spin down? central engine/black hole? CSM interaction? pair-instability?), along with other events that are the leading-edge science in transient astronomy.

**Comment by ethrane** [comment]
Tidal disruption events, super-luminous supernovae, the birth of magnetars... I think we are advocating for a lot of the same science, @jeffcooke. If I understand your most recent post, you would prefer to frame this science in terms of the extreme energy involved whereas Katie and I were imagining framing the same science in terms of stellar death. If we are, in fact, suggesting differently phrased questions to support the same science, then do feel free to post some alternative questions here. It could be useful to workshop some different phrasing!

**Comment by anaismoller** [comment]
I agree it is challenging to phrase it uniformly given that there is overlap between 
- stellar deaths
- stellar recycling (overlap with the question of element creation)
- extreme physics/densities

For the decade plan, I like "How do stars die?" 
"What are the physics in extreme matter densities?" could be linked to "What is the nature of matter at extreme densities? (proposed key question) #3" if we rephrase it as "What are the physics in extreme environments"

**Comment by jeffcooke** [comment]
@ethrane Yes, we are mostly talking about the some of the same events and, if interpreted in the broadest sense, could cover everything. But that will always seem like a stretch for any justification of science in the next decade needed beyond understanding the processes leading to the deaths of stars. Transients provide our only opportunity to study the physics of extreme energies and also the impact on their environment. Finally, some are non-stellar and unknown source explosions, so 'star' doesn't really seem to work. 
General comment regarding this question and some of the others. Maybe consider a *slightly* more scientific and complex wording for some of these, while keeping it broad and/or open-ended? I don't think there is a rule that it must be extremely simple. Trying to fit leading-edge complex science (square pegs) into simple few-word questions (round holes) for the report can be misleading and fall short of (and, importantly miss) what we need it to say for the next decade. If it's too simple, it can miss the point for many aspects of what the question needs to cover.

**Comment by ethrane** [comment]
@anaismoller, I like "What are the physics in extreme environments?" It's less neutron-star focused than the current phrasing––which has both pros and cons––but I think it's still relatively focused. I would be open to changing to this wording for https://github.com/ethrane/transients/issues/3 if folks prefer that.

@jeffcooke writes:
> General comment regarding this question and some of the others. Maybe consider a slightly more scientific and complex wording for some of these, while keeping it broad and/or open-ended? I don't think there is a rule that it must be extremely simple. 

In order to come up with these proposals, @kauchettl and I looked over the reports from the previous decadal plan as well as white papers from other countries. These short, snappy questions (followed by a one or two paragraphs of more technical text) seem to be pretty standard. Here are some examples of questions from previous reports:
- How do planetary systems form and evolve?
- What is the nature of the dark energy and dark matter?
- How did the first stars form?

That said, do please feel free to suggest alternative phrasing! If we have a few options, we might be inspired to combine phrases we like to get a wording that everyone really likes.

**Comment by nhurleywalker** [comment]
The way the question is currently phrased puts a big emphasis on cataclysmic events (supernovae, GRBs, mergers), and ignores some really interesting parts of the stellar life cycle that can also generate transient emission.

The study of radio stars is a burgeoning field that will be served well by ongoing surveys by the SKA precursors and of course with the SKA. Notably this also gives us unique ways to detect exoplanets.

After stellar death, compact stellar remnants are well-known to produce radio emission. Neutron stars have been a huge area of research for Australian astronomers over many years, through the discovery and timing of pulsars and magnetars. White dwarfs are now being found to emit pulsar-like radio emission, and the mysterious long-period radio transients look similar to pulsars, but their energetics are not explicable by pulsar mechanisms. These areas of research are enormously supported (at the very least) by CSIRO, Swinburne, USyd, and Curtin, and should be included somewhere in the TDA list of priorities. That could mean rephrasing this question to something like: "How do stars evolve and transform?" to indicate that they are and continue to be interesting before and well after "death".

**Comment by ethrane** [comment]
@nhurleywalker, I was just discussing this with @kauchettl...

We agree that there's a lot of interesting science associated with all phases of stars. However, we think the question, "How do stars evolve and transform?" is probably too broad, encompassing anything related to stars.

That said, things like pulsar-like emission from white dwarfs should be included under this question. Like black holes and neutron stars, white dwarfs are one of the remnants of stellar death. We see this question, in part, as using these remnants to study stellar death. 

We take your point that the blurb explaining what this question means should make it clear that we are not just interested in the cataclysmic moment of death, but on the remnants.

**Comment by nhurleywalker** [comment]
Well, I don't know how to more forcefully register that I really disagree with the way this question is currently phrased, and dislike it as a science question because it a) excludes huge swathes of exciting, current, and future Australian transient astronomy, and b) is still very vague about what it is actually asking -- we teach our 1st year undergraduates how stars die in the broadest sense, so it's not clear what science question is being asked here; c) stars really don't "die"; they really do just transform!

**Comment by GavinRowell** [comment]
What about "What are the violent processes during, and after a stars' life" ?

**Comment by GavinRowell** [comment]
Agree with @nhurleywalker that stellar flares in the radio via the SKA (& potentially gamma-rays via the CTA) from main sequence stars will become a future topic. Such flares can quickly inform us of the potential for stable environments on exoplanets.

**Comment by ethrane** [comment]
@GavinRowell, yeah, I've been mulling over a rewording along the lines you are suggesting to address Natasha's concern. Maybe something about "the death and afterlife of stars." Please keep the suggestions coming.

**Comment by jeffcooke** [comment]
Two things: Firstly, for this and several other issues, how important is the exact wording of the question as opposed to the description of what is covered under the question with regard to the decadal plan? Secondly, this is the time domain and multi-messenger astronomy WG. My understanding of that is that the WG focuses on things that change over time and messengers of that. We don't appear to be focusing on planet transits, stellar oscillations, variable stars or solar system objects. As such, that leaves transients. Essentially all transients fall under three categories: (1) the deaths of stars (via explosions, disruption, etc), (2) the 'second deaths' of stars (Type Ia, kilonovae, GW mergers, etc), or (3) some violent stellar, remnant, or AGN activity. As such, this decadal WG should first and foremost pose questions that address transients directly that we feel will be important for Australia during the next decade. It's pretty clear that this question, as written, falls short of addressing many transient types and those at all wavelengths, particles, and GWs. We should make sure that the questions put forth from this WG cover the transients that Australians work on, including what's needed for their detection and physical understanding. The field is broad, complex, and growing and it will likely take more than one question on this issue.

**Comment by taramurphy** [comment]
I quite like this question, or a variation of it. Clear and direct, and addresses physics.

**Comment by jeffcooke** [comment]
@taramurphy The question "How do stars die" may be clear and direct for stellar deaths, but not for any of the other equally important time domain events and transients as commented by @nhurleywalker and @GavinRowell and others. Hence my response. Either more than one question is needed or this question needs to be significantly rephrased to not omit all the other important transients, especially given that all topics that this WG covers will be expressed in a few (3-5) short questions. 

As such, one thing this question misses is any reference or implication on why we would study this topic. Why should the government spend money on understanding how stars die? In what way does it help astronomers, physicists, the broader community and even industry here on Earth? Examples of short and clear questions that do this would be two from above "What is dark matter and dark energy?" and "How do planets form and evolve?" They imply that there is a need to know these things and that there is a real use for this knowledge here on Earth. For us, the implications could come as understanding explosion mechanisms, high energies and magnetic fields, high densities, nucleosynthetic products, etc. with a rephrasing.
